### ISSUE TITLE
libappindicator is needed by stacer-1.0.7-1.x86_64

### ISSUE THREAD

**Issue by pitsoleil** [oguzhaninan/Stacer#57]
Hello, I tried to install it on centos but I have this error message 
Error: Package: stacer-1.0.7-1.x86_64 (/stacer-1.0.7)
           Requires: libappindicator
           Requires: libnotify

**Comment by oguzhaninan** [comment]
Hi,
You should install this packages.
[libappindicator](https://centos.pkgs.org/7/epel-x86_64/libappindicator-12.10.0-11.el7.x86_64.rpm.html)
[libnotify](https://centos.pkgs.org/6/centos-x86_64/libnotify-0.5.0-1.el6.x86_64.rpm.html)

**Comment by franshesco** [comment]
hello, I have exactly the same issue on centos 6.8. 

I have installed libnotify packages but it doesn't work for libappindicator. when i try 
`# yum provides **/libappindicator`
there is no match indicator. I have the last epel for centos 6.8 installed and when i try 
`# yum install libappindicator
Loaded plugins: fastestmirror
Setting up Install Process
Loading mirror speeds from cached hostfile
 * base: mirrors.linode.com
 * epel: mirror.sjc02.svwh.net
 * extras: mirrors.linode.com
 * updates: mirrors.linode.com
No package libappindicator available.
Error: Nothing to do
`
i have even installed the epel7 but still the same issue even if I downloaded the libappindicator rpm to install
